src/RpcServer/RpcServer.SmartContract.cs

```diff
@@ -69,7 +68,7 @@ private JObject GetInvokeResult(byte[] script, Signers signers = null)
                 Attributes = Array.Empty<TransactionAttribute>(),
                 Witnesses = signers.Witnesses,
             };
-            using ApplicationEngine engine = ApplicationEngine.Run(script, container: tx, gas: settings.MaxGasInvoke);
+            using ApplicationEngine engine = ApplicationEngine.Run(script, system.StoreView, container: tx, gas: settings.MaxGasInvoke);
```

Comment:
should pass the system.Settings too, otherwise it will be the default value null.

Reply:
Fixed.